<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startowe</class>
 <widget class="QDialog" name="startowe">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>1021</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz kontynent -&gt; kraj</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz date początkową</string>
   </property>
  </widget>
  <widget class="QPushButton" name="try_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>O K </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSpinBox" name="spinBoxMiesiac">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxRok">
      <property name="maximum">
       <number>2018</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz Chorobę</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combChoroba">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="obraz_choroba">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butt5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Wirus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butt6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bakteria</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butt7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pasożyt</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>294</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="butt1">
        <property name="text">
         <string>Afryka</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="butt3">
        <property name="text">
         <string>Australia</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="butt2">
        <property name="text">
         <string>Ameryki</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="butt4">
        <property name="text">
         <string>Eurazja</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="KrajeCombo"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
